core/json/src/main/scala/net/liftweb/json/JsonAST.scala

```diff
@@ -122,28 +208,44 @@ object JsonAST {
       case _ => false
     }
 
-    /** Return nth element from JSON.
-      * Meaningful only to JArray, JObject and JField. Returns JNothing for other types.
-      * <p>
-      * Example:<pre>
-      * JArray(JInt(1) :: JInt(2) :: Nil)(1) == JInt(2)
-      * </pre>
-      */
+    /**
+     * Return `n`th element from a `JArray` or `JObject` and return `JNothing` for any other kind of `JValue`.
+     *
+     * For example:
+     *
+     * {{{
+     * > val array = JArray(JInt(1) :: JInt(2) :: Nil)
+     * > array(1)
+     * JInt(1)
+     * }}}
+     */
     def apply(i: Int): JValue = JNothing
 
-    /** Return unboxed values from JSON
-      * <p>
-      * Example:<pre>
-      * JObject(JField("name", JString("joe")) :: Nil).values == Map("name" -> "joe")
-      * </pre>
-      */
+    /**
+     * Return unboxed values from JSON.
+     *
+     * For example:
+     *
+     * {{{
+     * > JObject(JField("name", JString("joe")) :: Nil).values
+     * Map("name" -> "joe")
+     * }}}
+     */
     def values: Values
 
-    /** Return direct child elements.
-     * <p>
-     * Example:<pre>
-     * JArray(JInt(1) :: JInt(2) :: Nil).children == List(JInt(1), JInt(2))
-     * </pre>
+    /**
+     * Return direct child elements of this `JValue`, if this `JValue` is a `[[JObject]]` or `[[JArray]]`.
+     *
+     * This method is useful for getting all the values of a `JObject` or `JArray` and will return them as a
+     * `List[JValue]`. If the `JValue` you invoke this method on is not a `JObject` or `JArray` you will instead
+     * get `Nil`.
+     *
+     * Example:
+     *
+     * {{{
+     * > JArray(JInt(1) :: JInt(2) :: Nil).children
+     * List(JInt(1), JInt(2))
+     * }}}
```

Comment:
Just remembered this—the cases where return values will diverge by the type of this `JValue` are good cases to use the `@return` tag, e.g.:

```scala
  /**
   * ...
   *
   * @return Direct children of this `JValue` if it is a `[[JObject]]` or `[[JArray]]`, or
   * `[[JNothing]]` otherwise.
   */
```